cognite_toolkit/cdf_tk/load.py

```diff
@@ -438,43 +516,66 @@ def load_resources(
 
     items: Iterable[Sequence[T_Resource]]
     if loader.load_files_individually:
-        items = ([loader.load_file(f, ToolGlobals)] for f in files)
+        items = [loader.load_file(f, ToolGlobals) for f in files]
```

Comment:
Changes this from a generator to a list, means we load everything and the `load_files_individually` does not load each file, upload it to CDF, before the next.